### ISSUE TITLE
Domain base / create first user

### ISSUE THREAD

**Issue by nuest** [hermzz/gestorneo#2]
I've been trying to run a local instance of gestorneo and started some rudimentary documentation, see https://github.com/nuest/gestorneo/blob/master/README.md

I got the database configured. Now, when I visit http://localhost/gestorneo/index.php I am forwarded to http://localhost/auth/login. Sso there seems to be a problem in operating under /gestorneo ?) Can I configure the root domain somehow?

Also, I was wondering how I can create the first (admin?) user - will it just work with the registration form?

Thanks for your help! I'm new to PHP...

**Comment by hermzz** [comment]
You've caught me on a trip so I won't be able to look at the issue of
working under /gestorneo until Tuesday.

As for the admin users, just register yourself as a regular user and then
go into the database and change the level field for your user to 'admin'.
We don't have a control panel for that yet.

**Comment by nuest** [comment]
No worries, I think it's a problem with my local setup anyway. I'll have to learn how to debug php applications myself.

Thanks for the pointer about the admin!

**Comment by hermzz** [comment]
Hey, sorry I took so long to get back to you, have you given it another try? Run into any other problems?

**Comment by nuest** [comment]
No, didn't find the time yet! Can you recommend a development environment, or should WAMP actually "just work" (as I would expect).

**Comment by layoutd** [comment]
Try XAMPP, pretty straightforward to set up.

As far as the route issue:
- Either set up a Virtual Host so you can use something like http://gestorneo.dev/
- Or, edit /application/config/config.php so that the $config['base_url'] reflects the correct path (and maybe put "index.php" in the $config['index_page'] a few lines down).

I think you have to manually create the admin user right now (using PHPMyAdmin or the like).

We should really consider upgrading to CI 2.13 which automatically detects the path, etc. I'll try to do that in the next couple of weeks/months/decades.

**Comment by hermzz** [comment]
I'm a big fan of virtual machines, that way you don't have to deal with all the windows shenanigans. If not, XAMPP is the way to go.

**Comment by hermzz** [comment]
Upgraded to 2.1.4 last night: 32e17b3b7d. At first glance it all looks like it's working, but there's probably things that have broken (I haven't tried adding content).
